### PR TITLE
[MNOE-579] Confirmation modal for requesting access

### PR DESCRIPTION
**Pull request by enizor**

### CODE REVIEW

**Comment by enizor** [comment]
rebased & used MnoConfirm. @alexnoox to review

File: src/app/components/mnoe-impersonate-button/mnoe-impersonate-button.coffee [hunk 1]
```diff
@@ -31,6 +31,17 @@
...
 
+    vm.openRequestModal = () ->
+      modalOptions =
+        closeButtonText: 'mnoe_admin_panel.dashboard.users.widget.local_list.modal.cancel'
+        actionButtonText: 'mnoe_admin_panel.dashboard.users.widget.local_list.request_access'
+        headerText: 'mnoe_admin_panel.dashboard.users.widget.local_list.modal.request_access_title'
+        bodyText: 'mnoe_admin_panel.dashboard.users.widget.local_list.modal.body'
+
+      MnoConfirm.showModal(modalOptions).then( ->
+        vm.requestAccess()
```
Comment (alexnoox) [inline src/app/components/mnoe-impersonate-button/mnoe-impersonate-button.coffee]: `vm.requestAccess()` should be passed as an `actionCb` in the `modalOptions`

Comment (enizor) [reply src/app/components/mnoe-impersonate-button/mnoe-impersonate-button.coffee]: mnoeImpersonateModal was the modal replaced by the use of MnoConfirm, just forgot to delete the files.

**Comment by enizor** [comment]
Code review addressed !

**Pull request merged**